tests/unit/lms/services/lti_grading/_v13_test.py

```diff
@@ -94,4 +118,4 @@ def response(self):
 
     @pytest.fixture
     def svc(self, ltia_http_service):
-        return LTI13GradingService(self.SERVICE_URL, ltia_http_service)
+        return LTI13GradingService("http://example.com/service_url", ltia_http_service)
```

Comment:
Can't be a `sentinel`. Needs to be a string. But we'd only use `SERVICE_URL` here